test/index.test.js

```diff
@@ -264,4 +264,19 @@ test('updates caniuse-lite for pnpm', async () => {
   )
 })
 
+test('updates caniuse-lite for bun', async () => {
+  let dir = await chdir('update-bun', 'package.json', 'bun.lockb')
+  match(
+    runUpdate(),
+    `Latest version:     ${caniuse.version}\n` +
+      'Updating caniuse-lite version\n' +
+      '$ bun update caniuse-lite\n' +
+      'caniuse-lite has been successfully updated\n'
+  )
+
+  let lock = (await readFile(join(dir, 'bun.lockb'))).toString()
+  ok(
+    lock.includes(`/caniuse-lite-${caniuse.version}`))
```

Comment:
Not sure if this check is OK. Maybe i's better to generate yarn.lock file and check versions in more readable way. https://bun.sh/docs/install/lockfile

Reply:
@kytta suggested using `pm ls -all` https://github.com/browserslist/update-db/pull/41

I will combine both your PRs and both of you will be mentioned in Cult of Martians.